internal/testcafe/config.go

```diff
@@ -57,7 +57,7 @@ type Suite struct {
 	DisableScreenshots bool              `yaml:"disableScreenshots,omitempty" json:"disableScreenshots"`
 	DisableVideo       bool              `yaml:"disableVideo,omitempty" json:"disableVideo"` // This field is for sauce, not for native testcafe config.
 	Mode               string            `yaml:"mode,omitempty" json:"-"`
-	Devices            []config.Device   `yaml:"devices,omitempty" json:"devices"`
+	Devices            []config.Emulator `yaml:"devices,omitempty" json:"devices"`
```

Comment:
A tad bit confusing internally, since it's actually simulators with regards to iOS 😅 However, testcafe is a whole different topic where I'd like to propose a more browser oriented approach (unrelated to your PR).

Reply:
@alexplischke this part was mainly to keep compatibility as it. I agree this is a bit misleading, and I forgot to mention that in the PR description.

I considered duplicating the `Device` object for Testcafe  in `testcafe` package to be more obvious. It may mean framework specific objects, but I'm not sure we want to go in that direction.
Other option would be to starting now, already introduce `config.Simulator` object and make it available for Testcafe. (We will still have the discrepancies between `Devices` field name and `config.Simulator` object).

We can speak of that on our next sync to take a decision.